examples/rdkafka_performance.c

```diff
@@ -131,8 +130,15 @@ uint64_t wall_clock (void) {
 }
 
 static void err_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque) {
-	printf("%% ERROR CALLBACK: %s: %s: %s\n",
-	       rd_kafka_name(rk), rd_kafka_err2str(err), reason);
+        if (err == RD_KAFKA_RESP_ERR__FATAL) {
```

Comment:
oh ok, i see, it makes a lot of sense because it's a global thing you want to get access to from anywhere, not just the context of where the error was first known. very good.

Reply:
It would still have been nice to use a bit to indicate an error is fatal rather than have a separate error code, but less important given the above.